contracts/pm/TicketBroker.sol

```diff
@@ -175,7 +176,7 @@ contract TicketBroker {
             sender.penaltyEscrow = 0;
         } else {
             amountToTransfer = _ticket.faceValue;
-            sender.deposit -= _ticket.faceValue;
+            sender.deposit = sender.deposit.sub(_ticket.faceValue);
```

Comment:
Due to the conditional on L171 (https://github.com/livepeer/protocol/pull/270/commits/db86d2bfb473523d8ada69ae2297bbe3ecf5df6d#diff-baed5767688fca63ab8b4efd937ce4ccR171) we know that this subtraction could never underflow so using SafeMath here just results in some extra unnecessary checks. I do see an argument for applying SafeMath everywhere for consistency sake. What do you think?

Reply:
I was prioritizing consistency :) If that's cool with you I'm happy to keep this change as-is. wdyt?

Reply:
Consistency works for me!